ui/.angular-ssl/server.key.pem

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
Why we insert private key in the repository?

Reply:
this is only developer key for having angular (front end) running on SSL (https)